tests/integration_tests/functional/test_rate_limiter.py

```diff
@@ -215,12 +210,11 @@ def _check_rx_rate_limiting(test_microvm, guest_ips):
 
     # First step: get the transfer rate when no rate limiting is enabled.
     # We are receiving the result in KBytes from iperf.
-    iperf_cmd = '{} {} -c {} -t {} -f KBytes -O {} -w {} -N'.format(
+    iperf_cmd = '{} {} -c {} -t {} -f KBytes -w {} -N'.format(
         test_microvm.jailer.netns_cmd_prefix(),
         IPERF_BINARY,
         guest_ips[0],
         IPERF_TRANSMIT_TIME,
-        IPERF_OMIT_TIME,
```

Comment:
Your commits look strange. Here you are removing omit time and in the next one you are adding it.

Reply:
I think you are looking at the commits in the wrong order. The first commit tries to make the iperf runs smoother - and also reduces omit time from 2s to 1s.

The second commit brings the long-duration fail-over test and removes omit time since it's no longer needed with the fail-over test.